client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -96,13 +106,37 @@ type ColumnConfig = Partial<ReturnType<typeof get_column_config_defaults>> & {
   formatter: FormatKey | ((val: CellValue) => React.ReactNode);
 };
 
+const get_default_filter_options_by_column = _.memoize(
+  (data: DisplayTableData[], col_configs_with_defaults: ColumnConfigs) =>
+    _.chain(col_configs_with_defaults)
+      .map((config, col_key) => ({ ...config, col_key }))
+      .filter((config) => config.is_searchable as boolean)
+      .map(({ col_key, plain_formatter }) => [
+        col_key,
+        _.chain(data)
+          .map(col_key)
+          .uniq()
+          .sort()
```

Comment:
Only just caught this, but we need to extract and reuse the display table sorting logic here. A searchable column might be providing a custom `sort_func`, or just rely on a `plain_formatter` (by default "plain" formatted values are used for sorting).

Take the covid measure column in any covid inforgraphic table. The filter dropdown doesn't appear sorted (at least not alphabetically) because the raw column data is actually the IDs and it's the formatter's job to supply plain language names.